CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+4.49.0
+==========
+### New Features
+- MSAL will now use `<region>.login.microsoft.com` when using regional ESTS-R. See [3252](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3252)
+- Added Support for Account Discovery when using Wam broker. See [3458](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3458)
+- Added support for app service system assigned managed identity (Managed Service Identity). See [3754](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3754)
```

Comment:
[Managed identities for Azure resources](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) provide Azure services with an automatically managed identity in Azure Active Directory. You can use this identity to authenticate to any service that supports Azure AD authentication, without having credentials in your code. MSAL .NET now supports acquiring token for Managed Identities for Azure App Service and Azure Virtual Machines. See [3754](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3754) and [3829](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3829).  Use `WithManagedIdentity` method on the `AcquireTokenForClient` API to get MSI token in MSAL. This is an experimental feature and may change in the future versions of MSAL